File: Converters/conv.py
#!/usr/local/bin/python
# -*- coding: utf-8 -*-

### BASE64
import base64
data = "U0hDe2lzc29fw6lfdW1hX2ZsYWd9"
dec = base64.b64decode(data)
print(dec)

### BASE32
data = "KNEEG63JNZUWG2LBNZSG6X3PL5TWC3LFPU======"
dec = base64.b32decode(data)
print(dec)

File: Converters/conv.rb
#!env ruby
# encoding: UTF-8

# Base64
require 'base64'
data = "U0hDe2lzc29fw6lfdW1hX2ZsYWd9"
plain = Base64.decode64(data)
puts "The plain text of #{data} is"
puts "#{plain}"

# Base32
require 'base32'
data = "KNEEG63JNZUWG2LBNZSG6X3PL5TWC3LFPU======"
plain = Base32.decode(data)
puts "The plain text of #{data} is"
puts "#{plain}"
